src/helpers.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
As `parsePack` relates directly to our use cases, my suggestion is to put it in `helpers.js` file.